test/specs/interceptors.spec.js

```diff
@@ -32,7 +32,8 @@ describe('interceptors', function () {
     axios.interceptors.request.use(function () {
       return {
         url: '/bar',
-        method: 'post'
+        method: 'post',
+        Promise: Promise
```

Comment:
The `config` that is passed to the interceptors already has all the defaults applied, so that shouldn't be a problem.